.buildkite/pipeline_trigger.sh

```diff
@@ -1,19 +1,23 @@
 #!/usr/bin/env sh
 
-if [[ "$BUILDKITE_MESSAGE" == *"[barebones ci]"* ]]; then
+barebones_description() {
   echo "Running barebones build due to commit messages"
   echo "Unit and static tests will be run"
   echo "Minimal instrumentation tests will be run"
   echo "End-to-end smoke tests will be run on minimum and maximum supported Android versions"
+}
+
+if [[ "$BUILDKITE_MESSAGE" == *"[barebones ci]"* ]]; then
+  barebones_description
 elif [[ "$BUILDKITE_MESSAGE" == *"[full ci]"* ||
   "$BUILDKITE_BRANCH" == "next" ||
   "$BUILDKITE_BRANCH" == "master" ]]; then
   echo "Running full build"
   echo "Unit and static tests will be run"
   echo "All instrumentation tests will be run"
   echo "All end-to-end tests will be run on all supported Android versions"
-  buildkite-agent pipeline upload .buildkite/pipeline.quick.yml
   buildkite-agent pipeline upload .buildkite/pipeline.full.yml
+  buildkite-agent pipeline upload .buildkite/pipeline.quick.yml
```

Comment:
This was needed because the steps are become reversed in the pipeline (because Buildkite inserts then in place).  As there are steps in full that depend on those in quick, full builds ended up being blocked from finishing.  I'll add a comment to that effect.